tests/e2e/e2e_test.go

```diff
@@ -94,7 +94,9 @@ func PodsStarted(t *testing.T, clientset *kubernetes.Clientset, namespace string
 		podUp[p] = 0
 	}
 
-	timeout := time.After(5 * time.Minute)
+	// Timeouts after 9 minutes if the Pod has not yet started
+	// 9 minute reasoning = 1 minute before 10-minute Golang test timeout.
+	timeout := time.After(10 * time.Minute)
```

Comment:
This `timeout` should be less than the entire golang test suite, which can be specified by flag to `go test -timeout`.

Reason being if this is less we fail knowing that pod never came up. But if golang test suite stops this test, the output is not very readable, so better we stop before test suite stops.

Reply:
But if we are considering on increasing the default time for running test of entire test suite then we are good to add this one.

Reply:
Yup. Since I added the other PR #263 this would be viable :+1: